include/SH3/system/glprogram.hpp

```diff
@@ -26,6 +27,7 @@ namespace sh3_graphics
     public:
 
         sh3_glprogram(const std::string& name) : programName(name){Load(name);};
+        sh3_glprogram(const std::string& name, std::vector<std::string> const &attribs) : programName(name){Load(name, attribs);};
```

Comment:
Hm. Is it even useful to have `Load` as a separate method?

I also just noticed that the `const` is written in different locations for both parameters. It'll have the same effect, but I'd prefer a consistent style.